src/components/progress/themes/linear/linear.progress.base.scss

```diff
@@ -1,21 +1,25 @@
+@use 'sass:math';
 @use 'styles/common/component';
 @use 'styles/utilities' as *;
 @use '../animations' as *;
 @use '../vars' as *;
 
 :host {
-    @extend %host !optional;
+    --track-size: #{rem(4px)};
+    --linear-animation-duration: 2000ms;
+}
+
+[part~='base'] {
+    @extend %base !optional;
```

Comment:
Why are you marking this as optional? Is there a case in which this placeholder class doesn't exist?